application/src/main/java/org/togetherjava/tjbot/CommandReloadUtil.java

```diff
@@ -0,0 +1,86 @@
+package org.togetherjava.tjbot;
+
+import net.dv8tion.jda.api.JDA;
+import net.dv8tion.jda.api.entities.Guild;
+import net.dv8tion.jda.api.requests.RestAction;
+import net.dv8tion.jda.api.requests.restaction.CommandListUpdateAction;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.togetherjava.tjbot.commands.BotCommand;
+import org.togetherjava.tjbot.commands.CommandVisibility;
+import org.togetherjava.tjbot.commands.system.CommandProvider;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.function.Predicate;
+import java.util.stream.Stream;
+
+/**
+ * Offers utility functions for reloading all commands.
+ */
+public class CommandReloadUtil {
```

Comment:
did we decide on whether to name utility classes `XXXUtil` or `XXXs`? Like `CommandReloads`? Have a quick check of the code base, to see what the standard was

Reply:
Looks to be pretty random and undecided, some are XXX, some XXXs, some XXXUtil(s)

Reply:
I see. Im voting for `CommandReloading`. I find it a bit confusing if the class is called like an utility class. Its not really an utility class, just a class with static methods only.